slave/process-voms.sh

```diff
@@ -42,17 +42,44 @@ function process {
 
 	# Iterate through every VO and check which user will be added or removed
 	for VO in `echo -e $VOS`; do
+		# First, check if some part should be skipped
+		if echo ${NO_CREATE_MEMBERS} | grep -E "([[:space:]]|^)${VO}([[:space:]]|$)"; then
```

Comment:
This can still causes trouble if $VO contains any character which will be interpreted as REGEXP special char, e.g. dot.

Something like this could work:
```bash
echo ${NO_CREATE_MEMBERS} | grep -E '([[:space:]]|^)'` echo ${VO} | sed -e 's/\./\\./g' `'([[:space:]]|$)'
```

or something like this:
```bash
echo "$VO" | grep -x -F "$NO_CREATE_MEMBERS"
```
but it requires that items in $NO_CREATE_MEMBERS will be separated by newline